docs/javascript/01-getting-started.md

```diff
@@ -17,14 +17,21 @@ The most basic usage is to request an instance of a class definition. TypeDI wil
 been created before and return the cached version or it will create a new instance, cache and return it.
 
 ```js
+import 'reflect-metadata';
 import { Container } from 'typedi';
 
+// require('reflect-metadata');
```

Comment:
Could you please clean this up a bit? Maybe stating to use either import or require. Also a whitespace is missing in `const { Container }= require('typedi');`

Reply:
Eh I'll just remove this then. There's no real way to make it look pretty without overloading this simple file.